### ISSUE TITLE
Clearing To-Do item's description doesn't work

### ISSUE THREAD

**Issue by intensecity** [RogerSelwyn/O365-HomeAssistant#215]
When editing a list item in Home Assistant, if one clears the Description field and hits Save it doesn't actually save the change. If one edits the item's name and/or due date, the name and/or due date change is saved, but not the update to the description.

As a workaround, I've been able to clear the Description by entering a single space or return character. Presumably, a Trim function is run on this field resulting in an empty value.

**Comment by RogerSelwyn** [comment]
Which service are you using to try to do the update `o365.xxx` or `todo.xxx`?

This may not be possible to solve, because the HA core does not seem to like passing through empty values. It maybe your workaround is the only solution.

**Comment by intensecity** [comment]
> Which service are you using to try to do the update
Sorry, I don't understand. I'm referring to using the built-in To-do lists applet in Home Assistant to managed a O365-synced To Do list.
My configuration.yaml file includes:
```
o365:
  accounts:
    - account_name: "NAME"
      client_id: "ID"
      client_secret: "SECRET"
      alt_auth_method: False
      enable_update: False
      todo_sensors:
        enabled: True
        enable_update: True
        track_new: True
```

> It maybe your workaround is the only solution.
I can live with that ^.^

Thanks for your help!

**Comment by RogerSelwyn** [comment]
Should be fixed for you in the latest release v4.7.1
